docs/source/bricks.rst

```diff
@@ -550,6 +593,96 @@ Examples:
   # Returns "Look at me, I'm flying!"
   extract_text_after(text, r"SPEAKER \d{1}:")
 
+``extract_email_address``
+----------------------
```

Comment:
Add a few `-`s here to get rid of this warning. The underline should cover the whole word:
```
``extract_email_address``
----------------------
/Users/mrobinson/repos/unstructured/docs/source/bricks.rst:597: WARNING: Title underline too short.
```